sourceror-driver/package-lock.json

```diff
@@ -0,0 +1,13599 @@
+{
```

Comment:
Not sure if you are using both `npm` and `yarn` as there are 2 files in this PR. It would be best to just manage a single lock file if possible, preferably `yarn`.